agent/credentials/manager_test.go

```diff
@@ -62,29 +63,23 @@ func TestGetTaskCredentialsUnknownId(t *testing.T) {
 func TestSetTaskCredentialsEmptyTaskCredentials(t *testing.T) {
 	manager := NewManager()
 	err := manager.SetTaskCredentials(TaskIAMRoleCredentials{})
-	if err == nil {
-		t.Error("Expected error adding empty task credentials")
-	}
+	assert.Error(t, err)
```

Comment:
It looks like `assert.Error` accepts the error message as well. Can we keep the error messages here and at other places where it was modified too? The messages serve the purpose of documentation as well and would be nice to have them around.